ui-admin/src/study/notifications/NotificationConfigView.tsx

```diff
@@ -29,77 +31,116 @@ export default function NotificationConfigView({ studyEnvContext }: {studyEnvCon
+      <div >
+        <label className="form-label">Notification type
+          <Select options={configTypeOptions} isDisabled={true}
+            value={configTypeOptions.find(opt => opt.value === config.notificationType)}/>
+        </label>
+      </div>
+      { isEventConfig && <div>
+        <label className="form-label">Event name
+          <Select options={eventTypeOptions} isDisabled={true}
+            value={eventTypeOptions.find(opt => opt.value === config.eventType)}/>
+        </label>
+      </div> }
+      { isTaskReminder && <div>
+        <div>
+          <label className="form-label">Task type
+            <Select options={eventTypeOptions} isDisabled={true}
+              value={taskTypeOptions.find(opt => opt.value === config.taskType)}/>
           </label>
         </div>
-        { isEventConfig && <div>
-          <label className="form-label">Event name
-            <Select options={eventTypeOptions}
-              value={eventTypeOptions.find(opt => opt.value === config.eventType)}/>
+        <div>
+          <label className="form-label">Remind after:
+            <div className="d-flex align-items-center">
+              <input className="form-control me-2" type="text" value={config.afterMinutesIncomplete}
+                onChange={e => setConfig(
+                  { ...config, afterMinutesIncomplete: parseInt(e.target.value) || 0 }
+                )}/>
+              minutes
+            </div>
           </label>
-        </div> }
-        { isTaskReminder && <div>
-          <div>
-            <label className="form-label">Task type
-              <Select options={eventTypeOptions}
-                value={taskTypeOptions.find(opt => opt.value === config.taskType)}/>
-            </label>
-          </div>
-          <div>
-            <label className="form-label">Remind after:
-              <div className="d-flex"> <input className="form-control" type="text" readOnly={true}
-                value={config.afterMinutesIncomplete}/> minutes</div>
-            </label>
-            <label className="form-label ms-3">Repeat reminder after:
-              <div className="d-flex"><input className="form-control" type="text"
-                readOnly={true} value={config.reminderIntervalMinutes}/>
-                minutes</div>
-            </label>
-            <label className="form-label ms-3">Max reminders:
-              <input className="form-control" type="text" readOnly={true} value={config.maxNumReminders}/>
-            </label>
-          </div>
         </div>
-        }
         <div>
-          <label className="form-label">Delivery
-            <Select options={deliveryTypeOptions}
-              value={deliveryTypeOptions.find(opt => opt.value === config.deliveryType)}/>
+          <label className="form-label">Repeat reminder after:
+            <div className="d-flex align-items-center">
+              <input className="form-control me-2" type="text" value={config.reminderIntervalMinutes}
```

Comment:
Could use `type="number"` here to get some stepper buttons.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number